planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/src/node.cpp

```diff
@@ -391,10 +397,10 @@ bool SurroundObstacleCheckerNode::checkStop(
 
 Polygon2d SurroundObstacleCheckerNode::createSelfPolygon()
 {
-  double front = front_overhang_ + wheel_base_;
-  double rear = -rear_overhang_;
-  double left = wheel_tread_ / 2.0 + left_overhang_ / 2.0;
-  double right = -(wheel_tread_ / 2.0 + right_overhang_ / 2.0);
+  const double front = vehicle_info_.max_longitudinal_offset_m_;
+  const double rear = vehicle_info_.min_longitudinal_offset_m_;
+  const double left = vehicle_info_.max_lateral_offset_m_;
+  const double right = vehicle_info_.min_lateral_offset_m_;
```

Comment:
`front` and `back` are identical to the old code, but I don't know why `left` and `right` had the `/ 2.0` at the end. The new version seems correct to me:
* `min_lateral_offset_m_` = `-(wheel_tread_m / 2.0 + right_overhang_m)`
* `max_lateral_offset_m_` = `wheel_tread_m / 2.0 + left_overhang_m`.

The names appear wrong to me too (`left` should be the negative number), but I left it like it was.

Reply:
> but I don't know why left and right had the / 2.0 at the end.

This is a bug and fixed in our latest code.

![image](https://user-images.githubusercontent.com/31987104/97515152-c347f980-19d3-11eb-9fb1-a718b7919af3.png)